docs/05-WP-Starter-Steps.md

```diff
@@ -54,49 +54,49 @@ Considering that `templates-dir` could be any folder, and considering that WP St
 
 This task will check that the WP Starter is able to reach required paths. It will check that WordPress and WP content folder exists and that the Composer autoload file is found.
 
-This ensures that subsequent steps will perform operation in right paths. Moreover, it ensures that at the moment WP Starter is performing its steps Composer have finished installing all dependencies.
+This ensures that subsequent steps will perform operations in the right paths. Moreover, it ensures that at the moment WP Starter is performing its steps Composer will have finished installing all dependencies.
 
 WP Starter will not proceed with other steps if this fails.
 
 ### `WpConfigStep`
 
-This is the main WP Starter step. It creates a `wp-config.php` that setups WordPress based on environment variables and adds WP Starter specific features as described in the *"WordPress Integration"* chapter.
+This is the main WP Starter step. It creates a `wp-config.php` that setups WordPress based on environment variables and adds WP Starter specific features as described in the [WordPress Integration](https://github.com/wunluv/wpstarter/blob/dev/docs/03-WordPress-Integration.md#cached-environment)  chapter.
```

Comment:
This points to _your_ fork, not to the source. Also, I did not add links because I don't want to point to `dev` branch